lib/app/views/test_details.erb

```diff
@@ -10,19 +10,12 @@
 <div class="modal-body">
   <div class="test-result-details">
 
-    <% unless @test_metadata[:description].nil? %>
+    <% if @test.description.present? %>
       <div class="test-result-details-description">
-        <%= markdown_to_html(@test_metadata[:description]) %>
-        <a class="test-result-details-url"target="_blank" href="<%= @test_metadata[:url] %>"><%= @test_metadata[:url] %></a>
-      </div>
-    <% end %>
-
-    <% unless @test_metadata[:details].nil? %>
```

Comment:
I didn't see anywhere where this would be populated.

Reply:
Anywhere where which would be populated? `@test_metadata[:details]`?

Reply:
Hmmm, there can be details on sequences (which there almost always are), and there can be details on result (which is an area that I would like to see improvement in... we just don't save enough from the assertions), but I'm not seeing anywhere that details could be on a test.  So I think this was a mistake or some kind of cruft left behind from a previous refactor.  I couldn't find anywhere to trigger this either.  So I'm good removing it.